src/c2_blockchain/p4_batched_extrinsics.rs

```diff
@@ -27,14 +27,14 @@ pub struct Header {
 // logic moves to the block level now.
 impl Header {
     /// Returns a new valid genesis header.
-    fn genesis() -> Self {
+    pub fn genesis() -> Self {
```

Comment:
Why was it necessary to make these `pub`? They are not `pub` in the previous parts.

Reply:
Using the Header::genesis() in tests and also useful for solutions